stdlib/TOML/test/print.jl

```diff
@@ -60,3 +60,14 @@ end
     d = TOML.parse(s)
     @test toml_str(d) == "user = \"me\"\n\n[julia]\n\n[option]\n"
 end
+
+@testset "special characters" begin
+    s = """
+    "\U1f355 \0 \x0 \x1 \t \b" = "\U1f355 \0 \x0 \x1 \t \b"
+    "\x7f" = "\x7f"
+    """
+    @test roundtrip(s)
+
+    d = Dict("str" => string(Char(0xd800)))
+    @test_throws ErrorException TOML.print(d)
```

Comment:
This appears to have introduced this defect in the test output:
```
TOML                               (41) |    31.70 |   0.67 |  2.1 |    1230.28 |  1353.48
      From worker 41:   str = "
```
```suggestion
    @test_throws ErrorException TOML.print(devnull, d)
```